firmware/Core/Src/antenna_deploy_drivers/ant_commands.c

```diff
@@ -0,0 +1,68 @@
+#include "main.h"
+
+#include "antenna_deploy_drivers/ant_commands.h"
+#include "antenna_deploy_drivers/ant_internal_drivers.h"
+#include "stm32l4xx_hal_i2c.h"
+#include "debug_tools/debug_i2c.h"
```

Comment:
Is the debug_i2c include required?

Reply:
I double-checked and found it wasn't useful. Removed that line from the file.